services/datamanager/builtin/capture_test.go

```diff
@@ -16,12 +17,26 @@ var (
 	enabledBinaryCollectorConfigPath            = "services/datamanager/data/robot_with_cam_capture.json"
 	infrequentCaptureTabularCollectorConfigPath = "services/datamanager/data/fake_robot_with_infrequent_capture.json"
 	remoteCollectorConfigPath                   = "services/datamanager/data/fake_robot_with_remote_and_data_manager.json"
-	emptyFileBytesSize                          = 30 // size of leading metadata message
+	emptyFileBytesSize = 30 // size of leading metadata message
 )
 
 func TestDataCaptureEnabled(t *testing.T) {
 	captureTime := time.Millisecond * 25
 
+	// On slower machines, it's possible that capture hasn't begun after captureTime. Give up to 20x
+	// as long for this to occur.
+	getCapturedFiles := func(captureDir string) []os.FileInfo {
```

Comment:
[supernit]: I think the terms getCapturedFiles and getAllFiles are a little bit confusing. As it stands, it kind of makes it sound like getCapturedFiles is a subset of getAllFiles, but IIUC getCapturedFiles is just getAllFiles with some waiting enforced.

Reply:
Perhaps `getAllFilesWithWait`?

Reply:
Definitely much more clear, updated